.github/workflows/tests.yaml

```diff
@@ -59,33 +59,36 @@ jobs:
           go get github.com/bazelbuild/bazelisk
           echo "$(go env GOPATH)/bin" >> $GITHUB_PATH
 
-      # Setup to the latest LLVM and Clang release.
-      #
-      # Ideally we would use the pre-installed versions in the image, but the
-      # debian packages for LLVM-15 currently don't work for fastbuild.
-      #
-      # For now, we rely on Homebrew to manage installing a correctly built
-      # toolchain. We also take some care to be as resilient as possible to
-      # issues fetching and installing the toolchain.
-      - name: Install Clang/LLVM using brew
+      # On macOS, use Homebrew to install a recent LLVM and Clang.
+      - name: Setup LLVM and Clang (macOS)
+        if: matrix.os == 'macos-latest'
         env:
           HOMEBREW_NO_INSTALL_CLEANUP: 1
           HOMEBREW_NO_INSTALLED_DEPENDENTS_CHECK: 1
-        # Use llvm@14 because llvm (15) has sanitizer issues.
         run: |
           echo '*** Updating brew'
           brew update
           echo '*** Installing LLVM deps'
-          brew install --force-bottle --only-dependencies llvm@14
+          brew install --force-bottle --only-dependencies llvm
           echo '*** Installing LLVM itself'
-          brew install --force-bottle --force --verbose llvm@14
+          brew install --force-bottle --force --verbose llvm
```

Comment:
I think we should explicitly force a version here, given that version changes require us to bump the cache version.

Should we / can we specify 15.0.1, or is @15 the best we can do?
Same question for the `llvm.sh` use below: can we force 15.0.1?

Reply:
brew doesn't support `@15`, only stable: https://formulae.brew.sh/formula/llvm

llvm.sh only supports major versions.